chains/orchestrator-paras/runtime/dancebox/src/tests/common/mod.rs

```diff
@@ -398,16 +396,6 @@ impl ExtBuilder {
         self
     }
 
-    pub fn with_safe_xcm_version(mut self, safe_xcm_version: u32) -> Self {
```

Comment:
this should not be removed afaik, dont we use it in any of the integration tests?

Reply:
We used it in 2 places: dancelight xcm as a part of ExtBuilder, that we replaced. And also in dancebox, I've also replaced fully ExtBuilder for dancebox, and removed it. 
In the other integration tests - we don't use it.